brigade/core/__init__.py

```diff
@@ -67,17 +83,46 @@ def __init__(self, inventory, dry_run, config=None, config_file=None,
         else:
             self.config = config or Config()
 
-        format = "\033[31m%(asctime)s - %(name)s - %(levelname)s"
-        format += " - %(funcName)20s() - %(message)s\033[0m"
-        logging.basicConfig(
-            level=logging.ERROR,
-            format=format,
-        )
+        self.configure_logging()
+
         if available_connections is not None:
             self.available_connections = available_connections
         else:
             self.available_connections = connections.available_connections
 
+    def configure_logging(self):
```

Comment:
Moving this message here to track it:

> @ktbyers said: IMO, we shouldn't enable this by default. People can enable it if they want to, but shouldn't default to enabled.

ok, let me revisit this.

Reply:
Yes, that message got hidden a bit because I made it on a commit that was changed in a later commit.

Reply:
Yeah, i agree with the `Config` object. I will leave it as it is right now and we can discuss what's the best behavior for this.